guides/doc-Managing_Hosts/master.adoc

```diff
@@ -55,6 +55,8 @@ include::common/assembly_host-status.adoc[leveloffset=+1]
 
 include::common/assembly_managing-packages.adoc[leveloffset=+1]
 
+include::common/assembly_package-mode-and-image-mode-hosts.adoc[leveloffset=+1]
```

Comment:
I think it's quite important and I'd include it in `common/assembly_overview-of-hosts.adoc`